redisinsight/api/migration/1755086732238-update-provider-names.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UpdateProviderNames1755086732238 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            UPDATE database_instance
+            SET provider = CASE provider
+                WHEN 'RE_CLOUD' THEN 'REDIS_CLOUD'
+                WHEN 'RE_CLUSTER' THEN 'REDIS_SOFTWARE'
+                WHEN 'REDIS_ENTERPRISE' THEN 'OTHER_REDIS_MANAGED'
+                ELSE provider
+            END
+            WHERE provider IN ('RE_CLOUD', 'RE_CLUSTER');
```

Comment:
`REDIS_ENTERPRISE` won't change to `OTHER_REDIS_MANAGED` since you filtered it out in `WHERE` section below